src/UI/Internal/Clickable.elm

```diff
@@ -0,0 +1,45 @@
+module UI.Internal.Clickable exposing (actionIcon, actionWrapElement, msgWrapElement)
+
+import Element exposing (Attribute, Element)
+import UI.Icon as Icon
+import UI.Link as Link
+import UI.RenderConfig exposing (RenderConfig)
+import UI.Utils.ARIA as ARIA
+import UI.Utils.Action as Action exposing (Action(..))
```

Comment:
looking at it again it seems like it's all about the icons. Doesn't it make sense to merge UI.Utils.Action and UI.Internal.Clickable under a single `UI.Icon.Action`?